docs/vidu-docs/图生视频.md

```
@@ -0,0 +1,93 @@
+> ## Documentation Index
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)